README.md

```diff
@@ -1,2 +1,27 @@
-# workloads
-Collection of snapshots of WebAssembly workloads
+# WebAssembly Workloads
+
+This repo contains a collection of snapshots of WebAssembly workloads
+served at [webassembly.github.io/workloads](https://webassembly.github.io/workloads).
+
+The goals of the repo are:
+* to provide a common place to share workloads between developers of
+  WebAssembly applications, engines and toolchain
+* to provide meaningful measurements of realistic paths through realistic
+  codebases (i.e., not collect synthetic benchmarks)
+* to contain builds that can both be run locally and served via
```

Comment:
That excludes non-Web implementations of WebAssembly. It would be nice to explicitly try to support them for at least a subset of workloads.

Reply:
There's two cases here:
* pure wasm workloads that could run in any wasm engine
* wasm workloads that specifically depend on a particular non-Web environment

The former would be useful to non-Web implementations but could also easily run in a browser with a trivial .html wrapper. Only the latter wouldn't, but such an environment doesn't exist yet, so I thought it made sense to require everything to be runnable in a browser at least for now; I can add a small note to this effect though.